user_guide/common_features.rst

```diff
@@ -0,0 +1,79 @@
+.. _user-guide-common-features:
+
+Common Features
+===============
+
+Customers, Channels and Accounts
+--------------------------------
+
+One of the main tasks for any CRM is ability to aggregate the details of a customer. 
```

Comment:
Edit: One of the main tasks of any CRM is the ability to aggregate the various details of a customer.